lib/tasks/set_parent_thumbnail.rake

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+namespace :hyku do
```

Comment:
@kirkkwang have you confirmed that running this tasks works?

Reply:
I have, but now that you mention it it was before a lot of the later changes so I will run it again and make sure it works still and update if needed.

Reply:
Still works!